db/v-3/core/init/_all_changelogs.xml

```diff
@@ -15,5 +15,7 @@
     <include file="lnk-project-github.xml" relativeToChangelogFile="true"/>
     <include file="comments.xml" relativeToChangelogFile="true"/>
     <include file="tag.xml" relativeToChangelogFile="true"/>
+    <include file="notification.xml" relativeToChangelogFile="true"/>
```

Comment:
why you add it here?

`init` was added for existed tables, for new one we can create a separate changelog file.
also, we can follow your suggestion and create a single changelog file for several tables (which are changed in scope of a single task)

Reply:
Removed the table from _init. As I understand it, we want to create new tables in separate files, but make changes in one common one?

Reply:
>  we want to create new tables in separate files
I didn't want (originally it was added for existed tables), but it sounds good.

So,
1. we create a single file for first creation table in `init`
2. all changes are in separate files in `db/v-3/<PART>/`